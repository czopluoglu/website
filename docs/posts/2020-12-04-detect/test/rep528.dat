00010000000010111000
10001101110111001110
10001100010101011111
11111101011011001101
10111100101110111101
00001110111001101111
11110011101110111111
11011001110001100011
01000001110010110111
11110111010011101111
11110011111101111011
01000111101101110110
10011100000101111111
10100100101001100000
11011000000111001111
10001101111111101010
10100001101011010101
01110101011110000110
00000101011001000101
10001000001000001010
10001101111111011111
10001101101010000010
00000000011010101110
10011011101010110110
11111111110100000010
11101110101111111101
01110101001010100110
10000100000100001110
11110111110101010101
10001001101001001110
11111011111100111011
00100000111011001011
10000000011111011111
10101101111101110101
11111010111110101100
01101111111111101111
01000011011110110111
11001101011100101111
10000100001000101011
00111011101001101010
10110001010000100110
10100001000111110111
00100100111110101110
01010001000001001010
10000011111110110111
00000011100000100110
11011111111000100111
11110110011111111111
11110111011100100100
00011111111010101010
01001001011010010100
11101010100000000010
10011101100000101110
00100111001110111000
11000100111011100110
00110011010110001110
01010001100010011100
11111111111100010111
10000111101010111100
11111100111110100110
01000000100110011110
11100001111011100111
10110111001110111111
10001001110101100110
11111111111111101111
10010101101110111111
10111100111100100101
00000101111001100110
10000000000000000010
00001101011011111111
00111000000110111111
00100001001101111011
00100101011111110100
10000010111110100011
00000101000011000111
11011100011111111111
11000101010001010111
11000100001110111111
00100101111000110011
11000101001000010111
00001011110011110111
11101111101000101110
10011101000110100000
10001000011011101111
00110001111100100110
00000001001100100110
00101110111100110011
10010110001010011010
10000100110010110111
00101000001010100010
01111010111010110011
10111100011011101101
10000110001110101010
10111101011111010011
11110011111111101110
10010001001010111001
00100110101010101100
10011010011010100000
01100101001111100110
11101100010001011111
11010101110000011111
11101110111110101110
00101011111100011011
01101101111111111000
00100101011011100111
00010001010010100010
01000010001111100001
10001101011110111111
10101000101011101011
00001001001010101111
01100010101010100000
00010000101000100010
00000111110111111000
01001011101100010111
11101101001011111001
10011010010111111010
11100111110011100101
11100111011111111111
01001000011000111100
01011011011101111001
10111111111111111011
10010111011111101111
01100001101100010110
00000000001001001110
11010000001110001110
11011111001011100111
10011001011111001011
11111001111111011110
10010111101010111110
11000011111111110010
00000010010000000110
10101101011011001010
11111111111101111011
01101101100010010011
01001101010110100011
11110111110101001111
00011100000011101011
00000010001101000100
11101011111111101111
10111011011010010010
00010110111001000111
10000001001111111111
10001001111111100111
11010110101110100011
11100101111010100110
01110101011000111000
10010011101100111110
10011001101010100011
10011001011011111110
00111001011000111011
01001001101111101111
00110001001100110110
01110000011100101111
10000111111001010111
11101111111001000010
10011001100111100000
11111111111011111011
11000011001100111111
00011011101000101011
00100001010100011001
01001000000000101100
11001001111111111110
00000001111001110011
00101000101100101100
01000011000010001011
11111010101110110010
01111011100011111100
10101011111001111001
10000100001100001100
10100000011010101111
10011101101100001111
10001001001001010001
10100000100010000100
00010011111010101111
00000001011010101110
00001000001001101101
11001101111010001011
11100011111101011111
01111111111110111010
11000100011111001101
11010010100111101101
10011100001001100010
00110001001010000010
01011001101001101111
00010101000110100110
00101000100000010011
01011101111001011111
11010110101100111110
10011100110110101111
11100100101010100010
10001001111010111111
01010000000000001110
10010011010000000010
11101010101010110101
11101111010000010110
11011101101010000011
11000101011110000111
10110101111100010100
11001110110011110111
10011111010010101100
10111100111010110100
10100000001111001111
00010000000010011110
10000000100001101011
10100001001110110110
11010010110100110000
10000001110100001111
11000001101110111101
01110111111111110010
01111111111111011011
01011000001010101100
00010000111110111111
00100000101010101110
01001011110000000111
11100011010001010011
00001010101111101011
01110101001111101110
00011101001000000000
11101111111100011111
00100000111101111111
00010110001000101100
10110001001011001100
00101001001001001111
01110111110111110111
01111001101000000011
01100110011011111110
11111011111111101111
10000101011101110110
11000001110110100001
00011001010000001101
11011001111000100111
01011111111110100110
01110100101010101010
01101101110000111110
00000000001000001010
01011001001010110111
11111011110101001011
01011111111011101111
10100111011110111010
10110111110010010101
01101101101100011010
11000000101010101000
00111111110111100001
11111001001000000010
10010100011010111111
11101100111011100110
01011111111111111111
10110111110100100110
10000111110001111010
01110110010011000101
