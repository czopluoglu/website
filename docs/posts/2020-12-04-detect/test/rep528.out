-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep528.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.82
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7959
                   IDN index value:   0.7053
                           Ratio r:   0.6462

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    2    2    3    1    2    3    2    3
    4    4    4    3    4    5    4    4    3    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    6

   -----------CLUSTER  2-------------

    2    3    4    7    9

   -----------CLUSTER  3-------------

    5    8   10   14   19   20

   -----------CLUSTER  4-------------

   11   12   13   15   17   18

   -----------CLUSTER  5-------------

   16
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+----++-++-+-++----
   +d+++----++----+--+-
   -+d++++++--+------+-
   -++d+++--+---++----+
   -+++d+-++------+++--
   --+++d+-++----+-+--+
   +-++-+d+++-+----+---
   +-+-+-+d++++++---+--
   --+-++++d++++------+
   ++-+-++++d+++-----++
   ++-----+++d-+----++-
   --+---++++-d++------
   +------+++++d----+++
   ---+---+---+-d-+++--
   +--+-+--------d-++++
   ++--+--------+-d+++-
   ----+++------+++d+--
   ----+--+--+-+++++d++
   -++------++-+-++-+d-
   ---+-+--++--+-+--+-d
 
 No cross validation for this DETECT run
